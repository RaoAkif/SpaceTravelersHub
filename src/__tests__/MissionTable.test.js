import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Table from '../pages/Missions/MissionTable';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Table />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
