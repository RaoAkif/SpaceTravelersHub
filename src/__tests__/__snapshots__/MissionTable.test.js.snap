// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container table-responsive"
>
  <table
    className="table-bordered table-striped table"
  >
    <thead>
      <tr
        className="fs-5"
      >
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
